### ISSUE TITLE
Run integration tests on 2 databases

### ISSUE THREAD

**Issue by dnavre** [sflpro/notifier#29]
After completing enhancement #28 and the tests being run on PostgreSQL docker. Add an additional integration test run on MySQL so both supported database engines are thoroughly tested in the integration test phase.

This basically means that each integration test will be run twice: once on PostgreSQL and afterward once more on MySQL.

I this the execution should go in the following steps.
1. Postgre is started
2. All integration tests are run on Postgre (no need to run unit tests)
3. Postgre is stopped
4. MySQL is started 
5. Continues as described above.

Basically, I want to avoid both engines being up at the same time. 

A possible problem are conflicts in the junit test report. All should work and be properly reported to sonar. The test cycles should not override each other's results.

**Comment by dnavre** [comment]
Done
